cs/test/compat/grpc/server/Program.cs

```diff
@@ -76,6 +76,8 @@ static void Main(string[] args)
 
             bool countdownSet = Countdown.Wait(30000);
 
+            pingServer.ShutdownAsync().GetAwaiter().GetResult();
```

Comment:
Consider using[`Task.Wait(TimeSpan)`](https://msdn.microsoft.com/en-us/library/dd235606(v=vs.110).aspx) so the test doesn't block here forever if the RPC methods never return for some reason.